fudo.c

```diff
@@ -28,12 +29,15 @@
 
 #define _XOPEN_SOURCE 600
 
+#include <assert.h>
```

Comment:
@FragmentedCurve By the way, I saw in my e-mail that you complained about the headers having being altered.
Nothing was effectively removed nor added, I just reorganized it in alphabetical order, like ``gofmt`` would.